core/src/main/kotlin/ch/ergon/dope/resolvable/expression/unaliased/type/arrayfunction/ArrayAppendExpression.kt

```diff
@@ -0,0 +1,17 @@
+package ch.ergon.dope.resolvable.expression.unaliased.type.arrayfunction
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.validtype.ArrayType
+import ch.ergon.dope.validtype.ValidType
+
+class ArrayAppendExpression<T : ValidType>(
+    array: TypeExpression<ArrayType<T>>,
+    value: TypeExpression<T>,
+    vararg values: TypeExpression<T>,
+) : ArrayFunctionExpression<T>("ARRAY_APPEND", array, value, *values)
+
+fun <T : ValidType> arrayAppend(
+    array: TypeExpression<ArrayType<T>>,
+    value: TypeExpression<T>,
+    vararg values: TypeExpression<T>,
```

Comment:
I think the same could be said here with Value, Values. Maybe a similar naming scheme you used before would make sense here: `value', `additionalValues', or something along those lines.

Reply:
done, thanks 😃